Makefile

```diff
@@ -326,13 +333,18 @@ define set-default-app-version
 endef
 
 update-helm-charts:
+	$(call update-helm-schemas, $(HELM_CHARTS_DIR))
 	$(call set-default-app-version,helm/scylla-operator,$(IMAGE_TAG))
 	$(call set-default-app-version,helm/scylla-manager,$(IMAGE_TAG))
 .PHONY: update-helm-charts
 
 verify-helm-charts: tmp_dir:=$(shell mktemp -d)
 verify-helm-charts:
-	cp -r helm/scylla-{operator,manager} '$(tmp_dir)'
+	cp -r helm/scylla helm/scylla-{operator,manager} '$(tmp_dir)'
+
+	$(call update-helm-schemas, $(tmp_dir))
```

Comment:
would it be better to have a separate targets and verify it by creating the schema file from scratch?

Reply:
That was the first approach I was going for but then `verify-helm-charts` compares the entire directories anyway. Should we just have a separate target that would compare the schemas first? Should `verify-helm-charts` exclude schemas?